app/Http/Controllers/JobController.php

```diff
@@ -150,4 +151,33 @@ public function acceptOrReject($userId, $jobId, $status)
             'users' => $users,
         ]);
     }
+
+    public function filter(Request $request)
+    {
+        if (is_null($request->tag)){
+            $jobs = Job::all();
+            foreach ($jobs as $job) {
+                $jobs->this = Job::findOrFail($job->id);
```

Comment:
`$jobs->this` là sao em nhỉ?
hàm `filter` này dùng để làm gì v?

Reply:
$job->this em làm bị thừa, em xóa đi r ạ. Hàm filter dùng để lọc công việc theo các Tag, Hàm này nhận vào id của các Tag và trả về các Job có chứa Tag đó